src/main/java/nl/altindag/log/LogCaptor.java

```diff
@@ -140,6 +141,11 @@ public static LogCaptor forName(String name) {
                 .collect(toList());
     }
 
+    public void addFilter(Filter<ILoggingEvent> filter) {
```

Comment:
The addFilter method is between `getLogEvents` and `toLogEvent`. getLogEvents method uses toLogEvent method and therefore I prefer to keep them together. Would it possible for you to move the addFilter method below toLogEvent?

Reply:
👍 Fixed

Reply:
Thank you very much for quickly adjusting it